src/twisted/conch/scripts/ckeygen.py

```diff
@@ -23,13 +24,25 @@
 
 
 
+supportedKeyTypes = dict()
+def _keyGenerator(keyType):
+    def assignkeygenerator(keygenerator):
+        supportedKeyTypes[keyType] = keygenerator
+        @wraps(keygenerator)
+        def wrapper(*args, **kwargs):
+            keygenerator(*args, **kwargs)
+        return wrapper
```

Comment:
I did a little digging here with the following snippets
```
supportedKeyTypes = dict()
def _keyGenerator(keyType):
    def decorator(method):
        supportedKeyTypes[keyType] = method
        @wraps(method)
        def wrapper(*args, **kwargs):
            print("foo")
            return method(*args, **kwargs)
        return wrapper
    return decorator
```

```
@_keyGenerator('rsa')
def generateRSAkey(options):
    # key generation code here
```
after executing these two code blocks `wrapper` method is not executed, and is executed only after an explicit call to `generateRSAkey` as following.

```
In [3]: generateRSAkey({'bits': '2048'})
Out[3]: foo

explicit call to keygenerator functions is currently missing in ckeygen hence the coverage issue, please guide how to proceed here